src/components/UI/Accordion/AccordionItem.tsx

```diff
@@ -1,27 +1,32 @@
 import React, { FunctionComponent, useState } from "react";
+import { addClassNameIfExist } from "../../../utils";
 
 type headingTag = "h3" | "h5";
 
 interface AccordionItemProps {
-  className?: string;
+  containerClassName?: string;
+  collapseClassName?: string;
+  contentClassName?: string;
   headingTag?: headingTag;
   heading: string;
   children: React.ReactNode;
 }
 
 export const AccordionItem: FunctionComponent<AccordionItemProps> = ({
-  className,
+  containerClassName,
+  collapseClassName,
+  contentClassName,
   headingTag = "h5",
   heading,
   children,
 }) => {
   const [open, setOpen] = useState(false);
   return (
-    <div className={`transition-color duration-200 mb-2${className ? ` ${className}` : ""}`}>
+    <div className={`transition-color duration-200${addClassNameIfExist(containerClassName, { prefix: " " })}`}>
       <div
```

Comment:
```suggestion
    <div className={`transition-color duration-200 ${addClassNameIfExist(containerClassName)}`}>
      <div
```

Reply:
if it doesnt exist the string will be `transition-color duration-200 ` -> whitespace here if u remove the prefix, but can la